packages/core/package.json

```diff
@@ -40,12 +44,12 @@
     "@stoplight/spectral-parsers": "^1.0.0",
     "@stoplight/spectral-ref-resolver": "^1.0.0",
     "@stoplight/spectral-runtime": "^1.0.0",
-    "@stoplight/types": "12.3.0",
+    "@stoplight/types": "13.0.0",
```

Comment:
Plans to bump to 13 in the other spectral packages?

Reply:
Yeah, I intend to do it